sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -10,6 +10,7 @@ and includes the main functions below:
 - Linked Entity Recognition
 - Healthcare Entity Recognition
 - Multiple Actions Analysis Per Document
+- Extractive Text Summarization
```

Comment:
it is not a function though

Reply:
We should mention that we have this feature. 

Trying to keep consistency with .NET and Python 
.net: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/textanalytics/Azure.AI.TextAnalytics
python:  https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/README.md

Reply:
Yeah let's try to be consistent but also correct. @maririos what do you think about updating the text above to `and includes the main features below`?

Reply:
sounds good with me!

Reply:
@kristapratico FYI Since we are changing the keyword `functions` to `features`.

Reply:
I made the change for JS in https://github.com/Azure/azure-sdk-for-js/pull/16789

Reply:
I got in this PR: https://github.com/Azure/azure-sdk-for-python/pull/20115/files